examples/browser-to-server/index.js

```diff
@@ -9,7 +9,7 @@ import { pushable } from 'it-pushable';
 let stream;
 const output = document.getElementById('output')
 const sendSection = document.getElementById('send-section')
-const appendOutput = (line) => output.innerText += `${line}\n`
+const appendOutput = (line) => output.innerHTML += `<div>${line}</div>`
```

Comment:
Why this change? Isn't this unnecessarily a XSS attack vector now? It's an example, yes. But people will copy this.

Reply:
Good call, this has refactored to be less permissive.